### PR TITLE
Add DO NOT MERGE label

### PR DESCRIPTION
**Pull request by Ana06**
I created a new label *DO NOT MERGE* as I think it is useful to indicate the state of PR. 

This commit add it to the *Labels for issues and PRs* section of the [contributing guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md#labels-for-issues-and-prs)

### CODE REVIEW

**Comment by differentreality** [comment]
I don't think we should open PRs that are not for merging. I believe we had this discussion before about PRs that said WIP in the title. Sure we talk about PRs and we update them, but when we reach the point of sending a PR, it should be mergeable. 

On a slightly different note, it seems that the contributing guide has gotten pretty big, making it harder for new people to read. I was thinking that we can move the new information into the wiki. What do you think?

**Comment by shlok007** [comment]
If not `do not merge` then maybe we could have a label like `needs another merge` / `needs more merging` for PRs that are ready to merge but needs more work probably on another PR to merge before it? One instance is: #1398 .

**Comment by differentreality** [comment]
Excellent point, that's another case I certainly do not want to encourage people to do! It is of very small importance to have open PRs that cannot be merged or reviewed,  as they depend on other PRs which we do not know if we are going to merge, or what changes there are gonna be. 

If you can't send a proper PR (which can be reviewed, and merged into master) then don't send it at all. You can work on something on your local machine, but if it depends on other features which have not been completed or decided yet, there is very little value in opening a PR. There might be a few cases that's an exception, but I don't want to encourage this workflow.

**Comment by differentreality** [comment]
@ChrisBr @hennevogel @lagartoflojo What do you think?

**Comment by Ana06** [comment]
@differentreality 

> when we reach the point of sending a PR, it should be mergeable.

It could happen that someone open a PR he thinks is mergeable and then when review it we prefer to merge another PR, and I think in that case it is important to make it clear.

**Comment by differentreality** [comment]
I agree on this. 
I just think that when this is the case, for some reason, we can leave a comment on the PR, and whomever merges a PR should do so responsibly (so at least read the last comment of the PR).

**Comment by lagartoflojo** [comment]
Another alternative is to require a Review Approval before merging anything. If the PR has no Approvals, then the "merge" button is disabled (this is configured in the repo admin page). So at least one of us has to review and approve the PR before merging. If the PR should not be merged for whatever reason, then just don't approve it.
I don't completely agree with @differentreality that a PR has to be created only when it's mergeable. Sometimes you just want to get the community's feedback, so it's still a work-in-progress.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -179,30 +179,33 @@ config.ichain_force_test_attributes = {:email => "testuser@example.com"}
 2. **Design**
   * Related to the looks and/or usability of the application; needs attention from someone who understands front-end and UX
   * If you are good with graphics and design, give it a shot!
-3. **Documentation**
+3. **DO NOT MERGE**
```
Comment (lagartoflojo) [inline CONTRIBUTING.md]: I don't think we really need numbering here. Maybe you could just replace all the `N.` with `*`.

**Comment by Ana06** [comment]
It seems that we have different points of views here, so I'll close this and remove the tag

**Pull request closed without merging**